src/admin/vote/Assign.jsx

```
@@ -412,6 +420,61 @@ export default function Assign() {
+                  ? results[choice.id] !== choice.selected[parseInt(value) - 1]
+                  : results[choice.id] === choice.selected[parseInt(value) - 1];
+              }
+              if (key === "selected") {
+                return isNegative
+                  ? !choice.selected.includes(value)
+                  : choice.selected.includes(value);
+              }
+              if (key.startsWith("selected[")) {
+                const index = parseInt(key.match(/\d+/)?.[0]) - 1;
+                if (isNaN(index)) return false;
+                if (!choice.selected[index]) return false;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)